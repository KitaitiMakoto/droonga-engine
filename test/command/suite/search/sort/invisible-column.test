#@include fixture/documents.jsons
{
  "type": "search",
  "dataset": "Droonga",
  "body": {
    "queries": {
      "result": {
        "source": "Sections",
        "sortBy": {
          "keys": ["-_key"],
          "offset": 2,
          "limit": 4
        },
        "output": {
          "elements": [
            "startTime",
            "elapsedTime",
            "count",
            "attributes",
            "records"
          ],
          "limit": -1,
          "attributes": ["title"]
        }
      }
    }
  }
}
