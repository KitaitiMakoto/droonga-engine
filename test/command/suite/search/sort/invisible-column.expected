{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "search.result",
  "body": {
    "result": {
      "count": 9,
      "attributes": [
        {
          "name": "title",
          "type": "ShortText",
          "vector": false
        }
      ],
      "records": [
        [
          "Groonga library"
        ],
        [
          "Geo-location (latitude and longitude) search"
        ],
        [
          "Sharable storage and read lock-free"
        ],
        [
          "Inverted index and tokenizer"
        ]
      ]
    }
  }
}
